dev/build/windows/makecoq_mingw.sh

```diff
@@ -1436,8 +1496,157 @@ function make_addon_equations {
+}
+
+# CompCert
+
+function make_addon_compcert {
+  make_menhir
+  make_menhirlib
+  # if build_prep http://compcert.inria.fr/release compcert-3.3 tgz ; then
+  if build_prep_overlay CompCert ; then
+    logn configure ./configure -ignore-coq-version -clightgen -prefix "$PREFIXCOQ" -coqdevdir "$PREFIXCOQ/lib/coq/user-contrib/compcert" x86_32-cygwin
+    log1 make
+    log2 make install
+    build_post
+  fi
+}
+
+# Princeton VST
+
+function install_addon_vst {
+    VSTDEST="$PREFIXCOQ/lib/coq/user-contrib/VST"
+
+    # Install VST .v, .vo, .c and .h files
+    install_rec compcert '*.v' "$VSTDEST/compcert/"
+    install_rec compcert '*.vo' "$VSTDEST/compcert/"
+    install_glob "msl" '*.v' "$VSTDEST/msl/"
+    install_glob "msl" '*.vo' "$VSTDEST/msl/"
+    install_glob "sepcomp" '*.v' "$VSTDEST/sepcomp/"
+    install_glob "sepcomp" '*.vo' "$VSTDEST/sepcomp/"
+    install_glob "floyd" '*.v' "$VSTDEST/floyd/"
+    install_glob "floyd" '*.vo' "$VSTDEST/floyd/"
+    install_glob "progs" '*.v' "$VSTDEST/progs/"
+    install_glob "progs" '*.c' "$VSTDEST/progs/"
+    install_glob "progs" '*.h' "$VSTDEST/progs/"
+    install_glob "veric" '*.v' "$VSTDEST/msl/"
+    install_glob "veric" '*.vo' "$VSTDEST/msl/"
+
+    # Install VST documentation files
+    install_glob "." 'LICENSE' "$VSTDEST"
+    install_glob "." '*.md' "$VSTDEST"
+    install_glob "compcert" '*' "$VSTDEST/compcert"
+    install_glob "doc" '*.pdf' "$VSTDEST/doc"
+
+    # Install VST _CoqProject files
+    install_glob "." '_CoqProject*' "$VSTDEST"
+    install_glob "." '_CoqProject-export' "$VSTDEST/progs"
+}
+
+function make_addon_vst {
+  if build_prep_overlay VST ; then
+    log1 make $MAKE_OPT
+    log1 install_addon_vst
+    build_post
+  fi
+}
+
+# ssreflect
+
+function make_addon_ssreflect {
+  if build_prep https://github.com/math-comp/math-comp/archive mathcomp-1.7.0 tar.gz ; then
+    cd mathcomp
+    log1 make $MAKE_OPT
+    log2 make install
+    build_post
+  fi
+}
+
+# coquelicot Real analysis
+
+function make_addon_coquelicot {
+  make_addon_ssreflect
+  if build_prep https://gforge.inria.fr/frs/download.php/latestfile/4132 coquelicot-3.0.2 tar.gz ; then
+    logn configure ./configure --libdir="$PREFIXCOQ/lib/coq/user-contrib/Coquelicot"
+    logn remake ./remake
+    logn remake-install ./remake install
+    build_post
+  fi
+}
+
+# extlib
+
+function make_addon_extlib {
+  if build_prep https://github.com/coq-ext-lib/coq-ext-lib/archive v8.8 tar.gz 1 extlib-v8.8 ; then
+    log1 make $MAKE_OPT
+    log2 make install
+    build_post
+  fi
+}
+
+# SimpleIO
+
+function make_addon_simple_io {
+  if build_prep https://github.com/Lysxia/coq-simple-io/archive master tar.gz 1 simple-io-master; then
+    log1 make $MAKE_OPT
+    log2 make install
+    build_post
+  fi
+}
+
+# Quickchick Randomized Property-Based Testing Plugin for Coq
+
+function make_addon_quickchick {
+  make_addon_ssreflect
+  make_addon_extlib
+  make_addon_simple_io
+  make_ocamlbuild
```

Comment:
I am experimenting with switching to opam. The main issue with opam is that it is tricky to get an "installation template" and executables with proper built in search paths. For developers I will provide a opam variant, since the path issue only exists for building installers. This will give me some means to judge long term maintainability of an opam solution.

Without using opam I don't see a way to reuse the information. I could extract it from opam files, but I don't think it is really worth the effort.